Feature/Education.feature

```diff
@@ -0,0 +1,20 @@
+﻿Feature: Education
+User wants to add education details into Profile Details
+Acceptance Criteria
+User is able to add the education details
+
+@mytag
+ Scenario: 01 Test Add Education sucessfully
+	Given User enter UniversityName as "Delhi University"

```

Comment:
The Given keyword is used for your test to be at a certain state. It's not really a "step" to enter fields.

Please read up on the Given keyword on the Gherkin/Specflow documentation